apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.test.js

```diff
@@ -922,7 +921,7 @@ describe('RegisterMultiPaymentService', () => {
                 number: billingAccount.number,
             })
             const [newReceipt] = await createTestBillingReceipt(admin, newBillingContext, newBillingProperty, newBillingAccount)
-            await updateTestAcquiringIntegration(admin, acquiringIntegration.id, { supportedBillingIntegrations: { connect: [{ id: newBillingIntegration.id }] } })
+            await updateTestAcquiringIntegration(admin, acquiringIntegration.id)
```

Comment:
why?)

Reply:
It does not matter now. There is a default group, so you shouldnt explicitly specify group of supportedAcquiringIntegration